VA-Explorer/va_explorer
# fixed location validation issues preventing user from saving changes …

fixed location validation issues that were adding faulty errors and preventing user from editing va records. 

To test: 
  import some va data, run the server and try editing a record (ex. minor tweak like first name change) and saving changes. You should get confirmation that the change went through
